intranet/static/css/bus.scss

```diff
@@ -145,3 +145,49 @@ span.bus-number {
         display: initial;
     }
 }
+
```

Comment:
Yea, I'm not sure exactly how to reconstruct it, what best practices should I be following here? @anonymoose2

Reply:
To start, this is actually SCSS, not CSS. You can nest selectors and it will combine them.

That means you can organize things hierarchically, just like in the accompanying HTML. In this case, that would be very helpful -- it's not at all clear what each of these selectors is doing.